.bazelrc

```diff
@@ -281,47 +280,53 @@ build:plain-fuzzer --linkopt=-fsanitize=fuzzer-no-link
 # Compile database generation config
 build:compdb --build_tag_filters=-nocompdb
 
-# Windows build quirks
+# Windows build behaviors
 build:windows --action_env=TMPDIR
 build:windows --define signal_trace=disabled
 build:windows --define hot_restart=disabled
 build:windows --define tcmalloc=disabled
-build:windows --define manual_stamp=manual_stamp
+build:windows --cxxopt="/std:c++17"
+# Build (compile) only statically linked objects
+build:windows --dynamic_mode=off
+# Force a monolithic -c opt envoy-static binary to include CRT and stdc++ libs.
+# This is not efficient or desirable behavior for fastbuild or //test/...
+# binary builds, but all components of envoy-static must be compiled /MT for
+# CRT static link, while binaries of //test/... using dynamic CRT must be compiled /MD
+build:windows --features=fully_static_link
 
-# Should not be required after upstream fix to bazel,
-# and already a no-op to linux/macos builds
-# see issue https://github.com/bazelbuild/rules_foreign_cc/issues/301
+# See issue https://github.com/bazelbuild/rules_foreign_cc/issues/301
+# Only windows requires these at compile time, no-op on linux/macos builds
 build:windows --copt="-DCARES_STATICLIB"
 build:windows --copt="-DNGHTTP2_STATICLIB"
 build:windows --copt="-DCURL_STATICLIB"
-build:windows --cxxopt="/std:c++17"
-
-# Required to work around build defects on Windows MSVC cl
-# Unguarded gcc pragmas in quiche are not recognized by MSVC
-build:msvc-cl --copt="/wd4068"
```

Comment:
Since the msvc-cl config option is now gone, we will have to remove `--config=msvc-cl` (replace with empty) from here: https://github.com/envoyproxy/envoy/blob/c8b894c558ddda6de7f1d7ffdab46216887c3db7/ci/windows_ci_steps.sh#L17

Reply:
Open question. If the underlying default flips to clang-cl we will need a config to pull it back. The absence of a rc line doesn't cause an issue. But maybe config msvc-cl should force-unset envvar USE_CLANG_CL?

Reply:
Yeah, that's a good point, possibly keeping `build:msvc-cl --action_env=USE_CLANG_CL=0` might be the safest also in case someone has it set in the environment?

Reply:
Adopted this thought, USE_CLANG_CL should be unset to '0' when invoking --config msvc-cl